src/app/proposals/components/ProposalStateAdmin.tsx

```diff
@@ -45,9 +58,9 @@ export const ProposalStateAdmin = ({ proposal }: Props) => {
     switch (proposal.status) {
       case PROPOSAL_STATUS.ACTIVE:
       case PROPOSAL_STATUS.PENDING:
-        return "This proposal still can be cancelled";
+        return "This proposal still can be cancelled by the admin.";
```

Comment:
"This proposal can still be cancelled by the admin" reads better to me, wdyt?

Reply:
Reads better to me as well! I will update it.